bigbluebutton-html5/imports/ui/components/video-preview/component.jsx

```diff
@@ -383,14 +391,18 @@ class VideoPreview extends Component {
   }
 
   displayPreview(deviceId, profile) {
-    const { changeProfile } = this.props;
+    const {
+      changeProfile,
+      skipVideoPreview,
+    } = this.props;
 
     this.setState({
       selectedProfile: profile.id,
       isStartSharingDisabled: true,
       previewError: undefined,
     });
     changeProfile(profile.id);
+    if (skipVideoPreview) this.handleStartSharing();
```

Comment:
```suggestion
    if (skipVideoPreview) return this.handleStartSharing();
```
to prevent us from running the rest of the function